src/acl/Options.cc

```diff
@@ -250,32 +237,26 @@ Acl::NoOptions()
     return none;
 }
 
-const Acl::ParameterFlags &
-Acl::NoFlags()
+const Acl::BooleanOption &
+Acl::CaseSensitivityOption()
 {
-    static const ParameterFlags none;
-    return none;
+    static const BooleanOption MyOption("-i", "+i");
+    return MyOption;
 }
 
 std::ostream &
 operator <<(std::ostream &os, const Acl::Option &option)
 {
-    if (option.valued()) {
-        os << '=';
-        option.print(os);
-    }
+    option.print(os);
     return os;
 }
 
 std::ostream &
 operator <<(std::ostream &os, const Acl::Options &options)
 {
-    for (const auto pos: options) {
-        assert(pos.second);
-        const auto &option = *pos.second;
-        if (option.configured())
-            os << pos.first << option;
-    }
+    for (const auto option: options)
+        os << *option;
```

Comment:
This will dump multiple options without any separator. Please take the opportunity to fix this old bug and ensure that 2+ options are whitespace separated.

Reply:
Just to clarify what happened here because this change request was addressed, but the changes addressing it are not visible in the above code: "Fixing this here" did not work as discussed in https://github.com/squid-cache/squid/pull/960#discussion_r782492794. The missing separator was then added to TypedOption::print() instead, addressing this change request, to the extent this out-of-scope request should be addressed in this PR (IMO).